src/styles.css

```
@@ -151,51 +151,63 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
-.be{background:rgba(26,86,219,.07);color:var(--blue);border-color:rgba(26,86,219,.2)}
-.bn{background:rgba(107,33,168,.07);color:var(--purple);border-color:rgba(107,33,168,.2)}
-.bh{background:rgba(197,48,48,.07);color:var(--red);border-color:rgba(197,48,48,.2)}
-.bm{background:#FFFAED;color:#92600A;border-color:rgba(251,191,36,.3)}
-[data-theme="dark"] .bm,[data-theme="dark"] .bam{background:rgba(251,191,36,.12);color:#FBBC04;border-color:rgba(251,191,36,.35)}
-.bc{background:rgba(0,135,90,.07);color:var(--green);border-color:rgba(0,135,90,.2)}
+.cat-pill{font-size:9px;font-weight:700;letter-spacing:.04em;text-transform:uppercase;padding:3px 8px;border-radius:6px;white-space:nowrap}
+.org-desc{font-size:13px;color:#52525B;line-height:1.5;margin-bottom:14px;}
+[data-theme="dark"] .org-desc{color:#A1A1AA;}
+.badges{display:flex;gap:6px;flex-wrap:wrap;margin-bottom:12px}
+.b{display:inline-flex;align-items:center;gap:3px;padding:3px 8px;border-radius:6px;font-size:11px;font-weight:600;border:1px solid transparent}
+.bv{background:rgba(194,65,12,.05);color:#C2410C;border-color:rgba(194,65,12,.1)}
```

Stylelint (17.11.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)